components/icon/doc/index.en-US.md

```diff
@@ -130,6 +130,19 @@ Let's assume that you deploy static assets under `https://mycdn.somecdn.com/icon
 
 Please call this in component's constructor or `AppInitService`.
 
+### Add Icons in Lazy-loaded Modules
+
+Sometimes, you want to import icons in lazy modules to avoid increasing the size of the main.js. You can use `NzIconModule.forChild`.
```

Comment:
```suggestion
Sometimes, you want to import icons in lazy modules to avoid increasing the size of the main.js. You can use `NzIconModule.patch`.
```